Examples/DeveloperCocoapodsExample/CocoapodsExample/Base.lproj/Main.storyboard

```diff
@@ -366,7 +366,7 @@
                                                 <subviews>
                                                     <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="DZM-Va-fhz">
                                                         <rect key="frame" x="20" y="0.0" width="374" height="345"/>
-                                                        <mutableString key="text">Embeds are a low-code pattern, requiring customer application development work to be done to create and expose injectable view areas where Appcues content can reside. This involves creating and registering a special AppcuesFrame view types with the iOS and Android SDKs, and including those views in the customer app layouts. This concept of view embedding is analogous to products like the Google Ads SDKs for mobile display ads. The details of how the embed views are registered with the native SDKs are outside of the scope of this document, which focuses on the data model updates. SDK documentation will cover those other integration topics.</mutableString>
+                                                        <string key="text">Embeds are a low-code pattern, requiring customer application development work to be done to create and expose injectable view areas where Appcues content can reside. This involves creating and registering a special AppcuesFrame view types with the iOS and Android SDKs, and including those views in the customer app layouts. This concept of view embedding is analogous to products like the Google Ads SDKs for mobile display ads. The details of how the embed views are registered with the native SDKs are outside of the scope of this document, which focuses on the data model updates. SDK documentation will cover those other integration topics.</string>
```

Comment:
I'm not sure why these changed from `mutableString` to `string`. I just opened the storyboard and added the button on the Events page. Xcode magic! 🙃

Reply:
ill just revert it.